lib/govobject/types/trigger.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var _ = require('lodash');
+var $ = require('../../util/preconditions');
+var GovObject = require('../govobject');
+var errors = require('../../errors');
+var inherits = require('util').inherits;
+
+/**
+ * Represents 'Trigger' Governance Object
+ *
+ * @constructor
+ */
+function Trigger(serialized) {
+  GovObject.call(this, serialized);
+}
+
+inherits(Trigger, GovObject);
+
+Trigger.prototype.dataHex = function() {
+
+  var _govObj = {
+    event_block_height: this.event_block_height,
+    payment_addresses: this.payment_addresses,
+    payment_amounts: this.payment_amounts,
+    proposal_hashes: this.proposal_hashes,
+    type: this.type,
+  };
+  // screwy data shims 'til we can fix this on dashd
```

Comment:
Actually this will change soon.

After 12.3 is released we should be able to add only the object and not the wrapper stuff, so just `_govObj` instead of inner/outer as defined here.